MasterSeed32.md

```diff
@@ -439,3 +439,29 @@ This is an example of a [Long MS32 String](#long-ms32-strings).
 
 * Secret share with index `S`: `MS100C8VSM32ZXFGUHPCHTLUPZRY9X8GF2TVDW0S3JN54KHCE6MUA7LQPZYGSFJD6AN074RXVCEMLH8WU3TK925ACDEFGHJKLMNPQRSTUVWXY06FHPV80UNDVARHRAK`
 * Master secret (hex): `dc5423251cb87175ff8110c8531d0952d8d73e1194e95b5f19d6f9df7c01111104c9baecdfea8cccc677fb9ddc8aec5553b86e528bcadfdcc201c17c638c47e9`
+
+# Appendix
+
+## Mathematical Companion
+
+Below we use the Bech32 character set to denote values in GF[32].
+In Bech32, the letter `Q` denotes zero and the letter `P` denotes one.
+The digits `0` and `2` through `9` do *not* denote their numeric values.
+They are simply elements of GF[32].
+
+The generating polynomial for our BCH code is as follows.
+
+We extend GF[32] to GF[1024] by adjoining a primitive cube root of unity, `ζ`, satifying `ζ^2 = ζ + P`.
+
+We select `β := G ζ` which has order 93, and construct the product `(x - β^i)` for `i` in `{17, 20, 46, 49, 52 77, 78, 79, 80, 81, 82, 83, 84}`.
```

Comment:
I believe this list should be `{17, 20, 46, 49, 52, 59, 77, 78, 80, 81, 82, 83, 84}`. That is, 59 should be there and 79 should not.

Reply:
Wait, lemme double-check that.

Reply:
79 should most definitely be there otherwise the code is broken.

Reply:
No, the existing text is correct.

Reply:
Though as a nit, you are missing a comma after `52`

Reply:
List is correct, but missing a comma.